### PR TITLE
[WIP] dynamic translation for file transfers and better "resume", "pause"

### PR DESCRIPTION
**Pull request by TheNain38**
Could you check if this is right before merging?

### CODE REVIEW

**Comment by TheNain38** [comment]
Wait there are bugs, I'm going to fix them, wait before merge

**Comment by agilob** [comment]
Still WIP? 

Could you change the last commit message to something more meaningful?

**Comment by TheNain38** [comment]
I'm going to refactor the code or it will get messy too quickly

File: src/chatlog/content/filetransferwidget.cpp [hunk 1]
```diff
@@ -100,6 +101,8 @@ FileTransferWidget::FileTransferWidget(QWidget *parent, ToxFile file)
     }
 
     setFixedHeight(78);
+
+    Translator::registerHandler(std::bind(&FileTransferWidget::retranslateUi, this), this);
```
Comment (tux3) [inline src/chatlog/content/filetransferwidget.cpp:105]: You need to call `Translator::unregister(this);` in the destructor, or after the file transfer widget dies the translator would be doing a use after free next time it translates.

Comment (TheNain38) [inline src/chatlog/content/filetransferwidget.cpp:105]: Done

**Comment by agilob** [comment]
@TheNain38 Still WIP?

**Comment by TheNain38** [comment]
I will try to finish it later

**Pull request closed without merging**